### DISCUSSION TITLE
In the second edition of the Book please use D12

### DISCUSSION THREAD

**Discussion by hindermath** [holgerflick/flixaccounting#19] [General]
For the work on a second edition of the book it would be cool if the pictures and descriptions of IDE settings could be made from RAD Studio 12 and DELPHI 12. Also, the source code could be adapted to D12. For example, the code of the `MessageDlg` could be use of the multi-line feature of D12.
At this time that can be looked like this:
```Delphi
    MessageDlg(
      '''
      Cannot launch application. Please check
      that database cnfiguration
      exists in settings.ini
      ''',
      TMsgDlgType.mtError,
      [mbOK],
      0
    );
```
After starting with an empty database section, the following Message was displayed, and the multi-line string was shown correctly as formatted in the above DELPHI code:
<img width="264" alt="image" src="https://github.com/holgerflick/flixaccounting/assets/681882/2cbd9a7a-7062-4273-9797-5b4889b32e25">

**Comment by holgerflick** [comment]
Great idea!

**Reply by holgerflick** [reply]
@hindermath  Please create a pull request for the message dialog.

**Reply by hindermath** [reply]
@holgerflick I have created an issue (#21) with my working code as a example that you can use if you wish. :-) I hope that is ok for you.

**Reply by holgerflick** [reply]
Fork this repo, change the code, branch, and create a pull request, please.

**Reply by hindermath** [reply]
Ok, working on it. I'm not absolute fine with git but I'm working on it so that we get this code into your mainrepo.

**Reply by hindermath** [reply]
Ok @holgerflick , I have created a pull request (#22) with the change from legacy string to multi-line string supported starting from D12.
